snap/hooks/post-refresh

```diff
@@ -2,3 +2,9 @@
 
 mkdir -p $SNAP_COMMON/logs
 cp $SNAP/docs/microovn/how-to/logs.txt $SNAP_COMMON/logs/README
+
+# If we upgraded from snap revision 379 (24.03.0+snap395808ff84) or earlier, ensure we move ovn.env to its own directory before running the start commands.
```

Comment:
nit: this comment becomes a bit out of place for the backport, but I guess it's not the end of the world either